components/playlist/browser/resources/components/playerControls.tsx

```diff
@@ -88,7 +88,7 @@ export default function PlayerControls ({ videoElement, className }: Props) {
     <Container className={className}>
       <div>
         <Control
-          iconName='start-outline'
+          iconName='previous-outline'
```

Comment:
need this file changes in this PR?

Reply:
This is part of the Leo bump - the icon was renamed so it won't build without it. The actual rename will happen in https://github.com/brave/brave-core/pull/19852